Core/Core.php

```diff
@@ -817,7 +813,7 @@ function create_function(string $args, string $code): false|string {}
  * by this function, the return value will be the string
  * Unknown.
  */
-#[Pure(true)]
```

Comment:
I suppose this change is also incorrect. The function operates on a resource handle and some of the resources can be managed globally such as database connections or network sockets. If a global resource pool is modified the resource type can change or become invalid. Also in case of this code it will return different results
```
$handle = fopen('example.txt', 'r');
echo get_resource_type($handle); // Outputs: stream

fclose($handle);
echo get_resource_type($handle); // Outputs: Unknown
```
Since the state of resource has been changed (resource connection closed)